ci/image_utils.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
```

Comment:
I believe you don't need a shebang for scripts that won't be directly executed.

Reply:
Looks like the lint is unhappy if I remove it:
```
Tips depend on target shell and yours is unknown. Add a shebang.
```